terraform_egress_pub/scripts/egress_pub.py

```diff
@@ -29,18 +29,19 @@
 HEADER = '''###
 # https://{domain}/{filename}
 # {timestamp}
-# DHS National Cybersecurity Assessments & Technical Services (NCATS)
+# CISA Vulnerability Management (VM)
 # {description}
-# Please contact ncats@hq.dhs.gov with any questions
+# Please contact vulnerability@cisa.dhs.gov with any questions
 ###
 '''
 
-# A list of dictionaries that define the files to be created and published.
-# When an ip is to be published, its assoicated application is compared to the
-# app_regex field.  If it matches it will be included in the associated
-# filename.
+# A list of dictionaries that define the files to be created and
+# published.  When an ip is to be published, its associated
+# application is compared to the app_regex field.  If it matches it
+# will be included in the associated filename.
 
-# The following CIDR blocks are where our test Qualys scans originate from:
+# The following CIDR blocks are where our test Qualys scans originate
```

Comment:
I don't think these are considered "test Qualys scans" anymore.  At this point, they can be referred to as "Qualys web application scans".

Reply:
See commit 54a3835.